doc/developer/webservices.rst

```diff
@@ -148,6 +148,50 @@ WMTS layer:
     }
 
 
+Autentication
+=============
+
+Login
+-----
+
+Used to login in the application.
+
+URL: ``.../login``
+
+Parameters (post form or query string):
```

Comment:
The password should not be sent as query string because this risks that passwords appear as plain text in log files.